test/SpecHelper.hs

```diff
@@ -54,25 +54,35 @@ getEnvVarWithDefault var def = do
   varValue <- getEnv (toS var) `E.catchIOError` const (return $ toS def)
   return $ toS varValue
 
+_baseCfg :: AppConfig
+_baseCfg =  AppConfig -- Connection Settings
+                      mempty "postgrest_test_anonymous" Nothing "test" "localhost" 3000
```

Comment:
No worries, it's all a preference thing. I was trying to get the comments to align. What's your preference? Perhaps a comment line between the type sig and definition?